OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -442,9 +463,19 @@ bool LoadAsset(IReadOnlyPackage package, string filename)
 					currentVoxel = world.ModelCache.GetModel(voxelName);
 					currentSprites = null;
 
-					// In case we're switching away from a type of asset that forced the music to pause.
+					// Just in case we're switching away from a type of asset that forced the music to pause.
 					Game.Sound.PlayMusic();
 				}
+				else if (allowedAudioExtensions.Contains(fileExtension))
+				{
+					// Pause music so it doesn't interfere with current asset.
+					Game.Sound.PauseMusic();
+
+					using (var soundStream = Game.ModData.DefaultFileSystem.Open(filename))
```

Comment:
This needs to be `prefix + filename`, otherwise the voices in TS's speech-gdi end up playing the nod versions.